### ISSUE TITLE
[BUG] Removing numbers from list doesn't work

### ISSUE THREAD

**Issue by jpelay** [hedyorg/hedy#3799]
**Describe the bug**
From level 4 to 11 removing a number from a list doesn't work, because the code we generate tries to find a string and not a number.

**Paste the Hedy code & level** 
Paste this on any level from 4 to 11:
```
ff is 1, 2
remove 1 from ff
remove 2 from ff
print ff at random
```
**Additional context:**

This is caused because this is the code we generate:

```
ff = ['1', '2']
try:
  ff.remove(1)
except:
  pass
try:
  ff.remove(2)
except:
  pass
```
As you can see, the remove tries to remove a number, not a string.

**Comment by imanshanawani** [comment]
Hi! Can I claim this bug?

**Comment by imanshanawani** [comment]
I am a student in a course working on a project and hoping to claim and fix this bug this weekend.

**Comment by Felienne** [comment]
Closed by #3834
